src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -231,11 +231,13 @@ ClientRequest::process_op(instance_handle_t &ihref, Ref<PG> &pg)
           return pg->with_locked_obc(
             m->get_hobj(), op_info,
             [this, pg, &ihref](auto obc) mutable {
-              return ihref.enter_stage<interruptor>(pp(*pg).process, *this
-            ).then_interruptible([this, pg, obc, &ihref]() mutable {
-              return do_process(ihref, pg, obc);
+              logger().debug("{}: got obc {}", *this, obc->obs.exists);
```

Comment:
```suggestion
              logger().debug("{}: got obc {}", *this, obc->get_oid());
```

Reply:
I see, `obc->obs` should print both `oid` and `exists` flag. Might be worth using it in order to have better context of the printed boolean in the logs.

Reply:
Oops, I guess I now recall ;-) why printed just `exists`:

```
../src/crimson/osd/osd_operations/client_request.cc:216:67:   required from here
../src/fmt/include/fmt/core.h:1757:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1757 |       formattable,
```

Sent a `fmt::formatter<...>` for `ObjectState` as a separated commit as it formally touches the classical thing.